### ISSUE TITLE
Win key modifier

### ISSUE THREAD

**Issue by gufpewga** [Belphemur/SoundSwitch#534]
Hello,
Could you allow Win key to be used as modifier?
I would like to use Win + C as shortcut. I disables Cortana so this shortcut is free.
But if I try to put Win key it's red and doesn't work.
I'm on latest verson to this date.

**Comment by Belphemur** [comment]
That's weird, Win key has been supported for a long time.
You need to press the Win key and the C key at the same time.

Can you try with different combination like WIN + M

**Comment by gufpewga** [comment]
Actually WIN + M is already in use on win10. But I tried without any success... If you want proof I can make screen capture.

**Comment by Belphemur** [comment]
I'm not able to reproduce your issue.

I can set it to WIN+N without problem or any other that isn't in use. It supposed to change to green when it's possible.

**Comment by bondt** [comment]
@Belphemur, but can you use `Win + C` as well? My guess is `Win + C` is still being caught by Windows, just like a lot of others listed on https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec (under "Windows logo key keyboard shortcuts").

**Comment by gufpewga** [comment]
I used microsoft PowerToys to remap shortcut Win+C into Ctrl+F12, then I used remaped shortcut inside SoundSwitch. Because I have Keychron keyboard and it has Cortana key that I use to switch output devices on my thinkpad. Plus thinkpad have shortcut key (star on F12) which I have set to Ctrl+F12. So now it works as expected... after long hassles :D
![image](https://user-images.githubusercontent.com/18383586/111275144-227e7180-8636-11eb-8347-752738e923e0.png)
